modular_citadel/code/modules/projectiles/guns/ballistic/handguns.dm

```diff
@@ -58,12 +58,9 @@
 
 //////10mm soporific bullets//////
 
-obj/item/projectile/bullet/c10mm/soporific
+/obj/item/projectile/bullet/c10mm/soporific
 	name ="10mm soporific bullet"
-	armour_penetration = 0
```

Comment:
these are definitely doing something

Reply:
Nope they have 0 ap its a subtupe of 10mm that has none thats a subtype of bullet that has none

Reply:
Yeah fairly sure this was unnecessary and was being inherited.